editor/src/clj/editor/workspace.clj

```diff
@@ -48,6 +48,8 @@ ordinary paths."
 ;; It's important to use the same class loader, so the type signatures match.
 (def ^:private ^DynamicClassLoader class-loader (DynamicClassLoader. (.getContextClassLoader (Thread/currentThread))))
 
+(defn get-class-loader [] class-loader)
```

Comment:
This shouldn't be a function because it does nothing, unlike `get-view-type`. You should make the `class-loader` var public by removing `^:private` metadata.